ext/pgsql/pgsql.stub.php

```diff
@@ -961,7 +961,7 @@ function pg_delete(PgSql\Connection $connection, string $table_name, array $cond
      * @return array<int, array>|string|false
      * @refcount 1
      */
-    function pg_select(PgSql\Connection $connection, string $table_name, array $conditions, int $flags = PGSQL_DML_EXEC, int $mode = PGSQL_ASSOC): array|string|false {}
+    function pg_select(PgSql\Connection $connection, string $table_name, array $conditions = [], int $flags = PGSQL_DML_EXEC, int $mode = PGSQL_ASSOC): array|string|false {}
```

Comment:
I should have told you, but you need to launch gen_stub.php (under build folder) on this file so the pgsql's C header get updated accordingly.